### PR TITLE
Create the contribution guide

### LINKED ISSUE #4: Write contribution guide
Ensure there are also instructions on how to create issues.

Check out our references for further inspiration: https://github.com/users/patdel0/projects/15?pane=issue&itemId=87300775

### PR DESCRIPTION
**Pull request by patdel0**
# Description

This PR introduces the initial draft of our contribution guide.

The plan is to keep updating it as the project evolves, addressing any
commonly asked questions and providing additional clarification as
needed.

The guide is inspired by the excellent contribution guide from Hack For
LA. It has been heavily simplified to better suit the goal of our
project of making contribution accessible to all, by attempting to not
overwhelm new developers with too much information.

In addition to the guide, a Pull Request template was added to streamline
the creation process.

Closes #4

## Type of change

- [X] This change requires a documentation update

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## 3. Fork the repository
+You can fork the repository by clicking <a href="https://github.com/patdel0/SAGP/fork">Fork</a>.  
+A fork is a copy of the repository that will be placed on your GitHub account.
+
+**Note:** It should create a URL that looks like the following -> `https://github.com/<your_GitHub_user_name>/SAGP`.  
+
+**For example:** `https://github.com/octocat/SAGP`.
+
+**Be Aware:** What you have created is a forked copy in a remote version on GitHub. It is not on your local machine yet.
```
Comment (matthiasjrichter) [inline docs/CONTRIBUTING.md:33]: For newer developers and for convenience, add a link on how to clone a repository. The official docs should be sufficient: https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository

**Review by matthiasjrichter — COMMENTED** [review]
Outstanding work!

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (patdel0) [reply docs/CONTRIBUTING.md:33]: Nice one! I'll add a section for that

Comment (patdel0) [reply docs/CONTRIBUTING.md:35]: Thanks for the suggestion. I added it under commit [4a0fc8a](https://github.com/patdel0/SAGP/pull/11/commits/4a0fc8a0b27dea617f4470f408a3e731398c9d7d).

**Review by matthiasjrichter — APPROVED** [review]
Looks great to me!

**Pull request merged**